src/devices/logitech_g535.c

```diff
@@ -163,6 +163,10 @@ static int g535_request_battery(hid_device* device_handle)
 static int g535_send_inactive_time(hid_device* device_handle, uint8_t num)
 {
     // Accepted values are 0 (never), 1, 2, 5, 10, 15, 30
+    if (num != 0 && num != 1 && num != 2 && num != 5 && num != 10 && num != 15 && num != 30) {
+        return HSC_OUT_OF_BOUNDS;
+    }
```

Comment:
Don't know if there is a cleaner way to check if `num` is one of the accepted values in C, normally I'd check if `num` is in an array of the accepted values but that looks very complicated to do in C :joy:

Reply:
I Will improve that today. It needs an error message with the correct values and a good way would be to round the user supplied value to multiple of 5 for num > 3 and then to use that. So that the user does not have to look up the correct values before that (except of maximum)